.github/workflows/tests.yml

```
@@ -57,16 +57,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest-8-cores]
+        os: [ubuntu-latest]
```

Let's leave this one for a faster feedback loop?

```suggestion
        os: [ubuntu-latest-8-cores]
```